tests/behat/features/bootstrap/WebFeatureContext.php

```diff
@@ -400,6 +402,10 @@ public function iShouldSeePrograms($arg1)
         $this->assertSession()->elementExists('css', '#mostViewed');
         $this->assertSession()->elementExists('css', '.program');
         break;
+      case 'scratchRemixes':
```

Comment:
please insert an empty line above to keep it consistent

Reply:
Done